schema.prisma

```diff
@@ -93,6 +93,7 @@ model Coordinator {
   beneficiaries Beneficiary[]
   campaigns     Campaign[]
 
+  @@unique([personId])
```

Comment:
well done with adding the unique index! please run "yarn prisma migrate dev" so that this generates the database change migration

Reply:
I think its ready.